lib/pages/settings/settings.dart

```diff
@@ -349,7 +349,8 @@ class GeneralConfigPage extends StatelessWidget {
               trailing: SizedBox(
                 width: 120,
                 child: RadioPicker<CommentSortType>(
-                  values: CommentSortType.values,
+                  values: CommentSortType.values
```

Comment:
Should we just remove that enum value completely (at least for now)?
It hasn't been fully implemented, so it'll just lead to other placeholder code crashing.

Reply:
The other related code was commented when we inherited lemmur. Also chat is coming and works on web so it's coming. I don't want to remove the enum value from our API client because that's the closest thing we have to source of truth for a backend